data/themes/darktable.css

```diff
@@ -876,8 +876,8 @@ entry
 #left #lib-plugin-ui scrolledwindow entry /* This will avoid having too much height on entry items in clone manager without touching other parts of the UI */
 {
   padding: 0px;
-  margin: 40px 3px;
-  min-width: 150px;
+  margin: 50px 3px;
```

Comment:
45px would be better. Test a portrait image and see icons and text around to see that (no text superpose). Good on landscape also with 45px.

Reply:
Works for me! Thanks.